fix: clarity in the tip for the specific source set (#3945)
JetBrains/kotlin-web-site: docs/topics/multiplatform/multiplatform-add-dependencies.md

If you want to use a multiplatform library just for specific source sets, you can add it exclusively to them. The
specified library declarations will then be available only in those source sets.

> > Use a general library name in such cases, not a platform-specific one. Like with SQLDelight in the example below, use `native-driver`, not `native-driver-iosx64` or other iOS versions. Find the exact name in the library's documentation.

Reviewer: ```suggestion
> Use a common library name in such cases, not a platform-specific one. Like with SQLDelight in the example below, use `native-driver`, not `native-driver-iosx64`. Find the exact name in the library's documentation.
```
